src/helpers/helpers.sh

```diff
@@ -36,7 +37,12 @@ source "$current_dir/environment.sh" "$@"
 source "$current_dir/configuration.sh" "$@"
 
 if [ "$(is_git_repository)" == true ]; then
+  temp_filename_suffix="$(git_root | tr '/' '_')"
+  export STEPS_FILE="/tmp/${PROGRAM}_${temp_filename_suffix}"
+  export UNDO_STEPS_FILE="/tmp/${PROGRAM}_undo_${temp_filename_suffix}"
   export INITIAL_BRANCH_NAME=$(get_current_branch_name)
+  export INITIAL_DIRECTORY=$(pwd)
+  export IN_SUB_FOLDER=$(is_in_git_sub_directory)
```

Comment:
This should either be before all the `INITIAL_...` or after. Based on ascii it appears it should go after
```
$ irb
2.2.0 :001 > "_" < "I"
 => false 
2.2.0 :002 > "I" < "_"
 => true 
2.2.0 :003 > 

```